README.md

```diff
@@ -58,57 +105,59 @@ these methods.
 Transformers take in a column and the meta data for that column as an input. Below we will
 demonstrate how to use a datetime transformer to transform and reverse transform a column.
 
-First we need to decompress the demo data included in the repository by running this
-command on a shell:
-
-```
-tar -xvzf examples/data/airbnb.tar.gz -C examples/data/
-```
-
-Afterwards, we can proceed to open a python interpreter and load the data
-
 ```python
->>> from rdt.transfomers import get_col_info
+>>> from rdt.transformers import get_col_info
```

Comment:
The `>>>` can be removed, and the inputs and outputs separated in different blocks, with some descriptive text in between.

Also, try to avoid creating unnecessary variables in the examples, and to use named arguments instead of positional ones. If lines are too long, fall back to vertical hanging indent format.

So, instead of

````
```python
>>> a_variable = 'variable contents'
>>> another_variable = 'very long variable contents'
>>> an_output = some_code(a_variable, another_variable)
>>> an_output
the output contents
```
````

Use:

````
```python
an_output = some_code(
    argument='variable contents',
    another_argument='very long variable contents'
)
```

The output will be "whatever it is":

```
the output contents, if needed
```
````

In this case, you would do:
````
```python
from rdt.transformers import get_col_info
column, column_metadata = get_col_info(
    table_name='users',
    col_name='date_account_created',
    meta_file='examples/data/airbnb/Airbnb_demo_meta.json'
)
```

The outputs are the column data, which is a `pd.Series` object:

```
0    2014-01-01
1    2014-01-01
2    2014-01-01
3    2014-01-01
4    2014-01-01
Name: date_account_created, dtype: object
```

And the column metadata, which is a dictionary with information about the column:

```
{
    'name': 'date_account_created',
    'type': 'datetime',
    'format': '%Y-%m-%d',
    'uniques': 1634
}
```
````